c/kastore.h

```diff
@@ -94,7 +94,8 @@ Unknown flags were provided to open.
 /** @} */
 
 /* Flags for open */
-#define KAS_READ_ALL            1
+#define KAS_READ_ALL            (1 << 0)
+#define KAS_GET_TRANSFERS_OWNERSHIP            (1 << 1)
```

Comment:
Might as well standardise on the "take" language, so how about "KAS_GET_TAKES_OWNERSHIP"